docs/cloud-native-security/environment-variable-capture.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[environment-variable-capture]]
+
+== Environment variable capture
+
+.Requirements
+[sidebar]
+--
+* This feature requires {stack} version 8.6 or higher.
+* In {stack} version 8.6, this feature is only available for Linux.
+--
+
+You can configure an {agent} policy to capture up to five environment variables ("env vars"). Captured env vars are associated with process events and recorded in the `process.env_vars` field.
+
+NOTE: Env var names must be 63 characters or fewer, and env var values must be 1023 characters or fewer. Values outside these limits are silently ignored.
+
+=== Setup
```

Comment:
This subheading doesn't seem necessary for a short, focused page that only has one procedure in it. But ignore this if the page is likely to expand soon with more subsections.